autobahn/wamp/interfaces.py

```diff
@@ -262,7 +262,10 @@ class ISession(object):
     @abc.abstractmethod
     def onConnect(self):
         """
-        Callback fired when the transport this session will run over has been established.
+        Callback fired when the transport this session will run over has
+        been established.
+
```

Comment:
Oh, while I'm thinking of it: do you prefer allowing ``None`` as callback or errback to ``_add_future_callbacks`` or should I mirror Twisted more and add ``_add_future_callback`` and ``_add_future_errback`` to the API as well?

For what it's worth, Twisted's ``addCallbacks`` allows a ``None`` errback but does not allow a ``None`` callback (so there'd be no way to add *just* an errback with such an API -- which is why I let them be ``None`` in the first place).